src/lib/profiles/service-provisioning/ServiceProvisioning.h

```diff
@@ -170,6 +181,7 @@ class ServiceProvisioningDelegate : public WeaveServerDelegateBase
     virtual WEAVE_ERROR HandleUpdateService(UpdateServiceMessage& msg) = 0;
     virtual WEAVE_ERROR HandleUnregisterService(uint64_t serviceId) = 0;
     virtual void HandlePairDeviceToAccountResult(WEAVE_ERROR localErr, uint32_t serverStatusProfileId, uint16_t serverStatusCode) = 0;
+    virtual void HandleIFJServiceFabricJoinResult(WEAVE_ERROR localErr, uint32_t serverStatusProfileId, uint16_t serverStatusCode) = 0;
```

Comment:
Very few devices will need this functionality.  So we should make this compile-time conditional on a WEAVE_CONFIG setting that defaults to off.

Reply:
Ok. Will add a compile flag for this.

Reply:
Done.